core/src/main/java/org/commonjava/maven/ext/core/impl/RelocationManipulator.java

```diff
@@ -203,9 +289,298 @@ private void updateDependencyExclusion( ProjectVersionRef depPvr, ProjectVersion
         }
     }
 
+    private PluginReference updatePluginGAVs( final Map.Entry<ConfigurationContainer, String> entry, final NodeList children )
```

Comment:
Naming : it seems we're not updating as such ; rather locating any plugins?

Reply:
It is finding a GAV tag set inside a plugin's configuration section.

Reply:
Changed name to `findPluginReference`.